webpack.config.js

```diff
@@ -0,0 +1,26 @@
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    filename: 'bundle.js',
```

Comment:
While If you'd met other requirements I'd have let this pass, but for the sake of meeting requirements of project, this should be name `main.js` as seen from this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/dist_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page) It actually mentions in the project description to have webpack.config match the one from the link.

This is not as important as other requirements in terms of name but would be good to follow.